editor/project_manager.cpp

```diff
@@ -1818,12 +1832,35 @@ ProjectManager::ProjectManager() {
 	tabs->add_child(tree_hb);
 
 	VBoxContainer *search_tree_vb = memnew(VBoxContainer);
-	search_tree_vb->set_h_size_flags(SIZE_EXPAND_FILL);
 	tree_hb->add_child(search_tree_vb);
 
+	HBoxContainer *sort_box = memnew(HBoxContainer);
+	Label *sort_label = memnew(Label);
+	sort_label->set_text(TTR("Sort:"));
+	sort_box->add_child(sort_label);
+	sort_box->add_spacer(true);
+	Vector<String> vec1;
```

Comment:
Please give the variable a clearer name. `filters` seems like a natural fit, or `filter_names`.

Reply:
I see there's another one later on, so maybe `sort_filters` here and `search_filters` for the other.